packages/helpers/src/dom/isDescendant.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-cond-assign */
+export default function isDescendant(el, attr, value) {
+  const doesElementHaveAttribute = el => el?.getAttribute(attr) === value;
+  let isChild = doesElementHaveAttribute(el);
```

Comment:
this function tries to find the parent of a node child but does it with a too specific method IMHO. 

Maybe doesn't worth having it at a helper package level.

Also worth noting that in the next line

in this line https://github.com/acl-services/paprika/pull/984/files#diff-fc157e57b35fbc14e972a59db1410d976b2f4c7bea94f3d29be07751b64ba690R115

you are assuming that only one overlay will ever exist in a webpage, if that is the case then there is no case for this method.

```
document.getElementById('overlay').contains(element)
```

Will find if the dom is a descendant of an overlay.

I would suggest that maybe you can use an anchor instead of an id.

`data-pka-anchor="overlay"`

```
document.querySelector("[data-pka-anchor='overlay']").contains(element)
```

Reply:
Sidenote: Because the TakeOver overrides the Paprika Overlay props, which includes the `data-pka-anchor` I decided not to use it (https://github.com/acl-services/paprika/blob/master/packages/Takeover/src/Takeover.js#L66) and used an `id`.